react-frontend/src/App.js

```diff
@@ -1,48 +1,48 @@
-import React, { Component } from 'react';
-import Swimlane from './Swimlane.js'
+import React, {Component} from "react";
+import Swimlane from "./Swimlane.js";
+import TaskData from "./model/TaskData";
+import SwimlaneData from "./model/SwimlaneData";
 
 export default class App extends Component {
 
     constructor() {
         super();
+        let tempSwimlaneData = new Map();
+        tempSwimlaneData.set('Gauss', new SwimlaneData('Gauss', [
```

Comment:
It'll be removed anyways (as initially we would like to get all those data from backend) so from the assumption it's temporary and shouldn't be here at first place (if you still don't like it)